### ISSUE TITLE
Bulk Delete - is this still plugin supported?

### ISSUE THREAD

**Issue by bceluck** [sudar/bulk-delete#763]
I had purchased a customization and I sent an email and have not got a response. I also submitted a pre-sales question using your form submission and using an email to support@bulkwp.com and not gotten a response. I don't want to purchase a plugin that doesn't have support. Thank you

**Comment by chmtt** [comment]
I purchased an addon and have the same issue. I got no bill and the support isn't available.

**Comment by bceluck** [comment]
Ya, I gave up on these guys. Going to purchase a different delete add-on. Thanks for letting me know.

**Comment by gcwebsites** [comment]
I've just bought two add-ons with no problems......

**Comment by yumaheymans** [comment]
Bought a license, doesn't work. I don't get a response from any of the contact channels. **Don't buy**.

**Comment by jwopus** [comment]
6 emails in 2 weeks might seem like a lot — but alas I'm getting zero response… I found that bulk delete by custom field doesn't respect the "date before" values.

**Comment by gcwebsites** [comment]
> 6 emails in 2 weeks might seem like a lot — but alas I'm getting zero response… I found that bulk delete by custom field doesn't respect the "date before" values.

Yes, I think there are some bugs. You would be best to write your own script to delete stuff, that is what I did for some items.

I don't think the plugin is maintained any further.
